### ISSUE TITLE
Is there a way to see item drops with game minimized?

### ISSUE THREAD

**Issue by MrBeatDown** [kolton/d2bot-with-kolbot#2004]
Bot all setup running everything I want almost flawlessly. The only thing I could not figure out, or maybe its not a feature, is how to see items that the bot picks, keeps, and stashes, while the game is minimized. The bot interface console is just showing me the fail scripts that happen sometime, or fail to join games. Was hoping to be able to see items picked. Is this possible?

Also, I could not find a link to maybe some in game commands to stop the bot, then restart it. That way if I do find something good, I can pause, list it for sale, then resume game. 

Thanks for helping.

I finally found something that was logged. It actually logged a small charm but it is not logging the Keys or the Gul rune. Is there some kind of setting in my character config script that needs edited?

**Comment by alfton** [comment]
yes, you have a log file you can open

**Comment by MrBeatDown** [comment]
> yes, you have a log file you can open
> the file Im talking about you find under /trunk/d2bs/d2bs.txt file. there you see all the log, of error, what script, and what items..

Hey thanks, Yeah I just figured it out. Was hoping that it would show it on the console so I can just mouse over the image to see what was being sold to the bot. I basically scrapped the default pickit and made my own. Appears to be A ok and pass the nipcheck addon. I still get worried that I don't pick and keep what I want so I wanted to see it.


But that has been resolved now and thank you for replying. Up next I will definitely be trying to make a script to run the Claw Viper Temple. Cannot believe one is not here. That is the best MF in the game..... Especially for a zealot bot like im running. Will get to work on it soon. Reset is coming up though and that will be taking most my spare time.
